js/src/ScatterGL.ts

```diff
@@ -1255,7 +1230,7 @@ export class ScatterGL extends Mark {
 
     elements_added
       .append('path')
-      .attr('transform', (d, i) => {
+      .attr('transform', function (d, i) {
```

Comment:
```suggestion
      .attr('transform', (d, i) => {
```

(applies multiple places in the file)

Reply:
I was planning not to touch too much on the other files, but I can do this one!

Reply:
I mean, actually, it's a bunch of changes. Sorry, I was planning to do this kind of thing in the actual refactoring of the file... I can remove the changes I did here, I was changing a variable name only ;/
Other than that, I think it's ready for a new review.